src/test/java/org/apache/commons/lang3/RandomStringUtilsTest.java

```diff
@@ -511,7 +511,7 @@ public void testRandomStringUtils() {
     public void testRandomStringUtilsHomog() {
         final String set = "abc";
         final char[] chars = set.toCharArray();
-        String gen = "";
+        String gen;
```

Comment:
Again, if you're doing this move the declaration into the fior loop; don't leave variables undeclared as it's bug prone